### DISCUSSION TITLE
Cable requirements for display?

### DISCUSSION THREAD

**Discussion by Chipthemunk** [tjko/fanpico#52] [Q&A] [answered]
Hello,
I would like to mount the display externally on my case and have fanpico inside. Is there anything I need to look out for or can I use whatever? Length will be between 30 and 60 cm. Thank you.

**Comment by tjko** [comment, marked as answer]
OLED (I2C) should be fine I would think. I've only tested with about 30cm "Dupont" cable and it seemed to work fine. For longer connection using shielded cable might help, if there is issues.  I'll see if I can find longer cable to test with...

With LCD (SPI), I have a Waveshare display working with about 20cm cable. With longer cable there started to be signal integrity issues. Again shielded cable might help or lowering the SPI bus clock speed.

![lcd_cable](https://github.com/tjko/fanpico/assets/1075726/59a3626e-1b1e-44bf-acc6-2e8d5eb3611c)
![lcd_connected](https://github.com/tjko/fanpico/assets/1075726/783f177f-bb19-4aa0-aa5b-057fc3a1d170)
